src/NuGet.Services.BasicSearch/ServiceEndpoints.cs

```diff
@@ -116,6 +100,17 @@ private static bool GetIncludePrerelease(IOwinContext context)
             return includePrerelease;
         }
 
+        private static bool GetIncludeSemver2(IOwinContext context)
+        {
+            System.Version semVerLevel;
+            if (System.Version.TryParse(context.Request.Query["semVerLevel"], out semVerLevel))
```

Comment:
The values for the parameter names ("`semVerLevel`", "`luceneQuery`", etc) should be in a constant.

Also you could probably replace these methods with a generic helper function (`static T GetParameter<T>(IOwinContext context, string name)`) using [Convert.ChangeType](https://msdn.microsoft.com/en-us/library/dtb69x08(v=vs.110).aspx), but I could understand if that would be too much refactoring for this PR.